src/test/java/com/ksm/wstbackoffice/controller/CountryControllerIT.java

```diff
@@ -37,6 +40,23 @@
         );
     }
 
+    public static Stream<Arguments> badArgumentsProvider() {
+        return Stream.of(
+                Arguments.of(Arrays.asList(), 400),
```

Comment:
In Java 11 you can use List.of(bla1, bla2)
To create an empty list just use Collections.emptyList() or List.of()